cmd/conduit/root/version/version_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2025 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package version
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/conduitio/ecdysis"
+	"github.com/matryer/is"
+)
+
+func TestVersionCommandExecute(t *testing.T) {
+	is := is.New(t)
+
+	buf := new(bytes.Buffer)
+
+	out := &ecdysis.DefaultOutput{}
+	out.Output(buf, nil)
+
+	cmd := &VersionCommand{}
+	cmd.Output(out)
+
+	expectedOutput := fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH)
```

Comment:
is there a reason why we're not using `conduit.Version` here too?

Reply:
@maha-hajja you're right. Just updated it https://github.com/ConduitIO/conduit/pull/2104/commits/316c03470e072942fb0616745ad52be0476d620a, thank you